Realm/Realm.Fody/Common/RealmWeaver.cs

```diff
@@ -405,32 +406,21 @@ private WeavePropertyResult WeaveProperty(PropertyDefinition prop, TypeDefinitio
                 switch (collectionType)
                 {
                     case RealmCollectionType.IList:
-                        var concreteListConstructor = _references.System_Collections_Generic_ListOfT_Constructor.MakeHostInstanceGeneric(elementType);
-
-                        // weaves list getter which also sets backing to List<T>, forcing it to accept us setting it post-init
-                        if (backingField is FieldDefinition backingListField)
-                        {
-                            backingListField.Attributes &= ~FieldAttributes.InitOnly;  // without a set; auto property has this flag we must clear
-                        }
-
                         ReplaceCollectionGetter(prop, backingField, columnName,
-                                            new GenericInstanceMethod(_references.RealmObject_GetListValue) { GenericArguments = { elementType } },
-                                            concreteListConstructor);
+                                            new GenericInstanceMethod(_references.RealmObject_GetListValue) { GenericArguments = { elementType } });
                         break;
                     case RealmCollectionType.ISet:
-                        _logger.Warning($"{type.Name}.{prop.Name} is of type ISet which is not officially supported yet. Some functionality may not exist yet or there may be bugs/known issues that can result in undefined behavior, including data loss. Official support for the datatype will come in a future Realm release.");
+                        Debugger.Break();
```

Comment:
Reminder to remove this when the PR is about to be merged.

Reply:
🤦‍♂️ Thanks for spotting it